local/proxy/cert_store.go

```diff
@@ -23,20 +23,20 @@ import (
 	"crypto/tls"
 	"sync"
 
-	lru "github.com/hashicorp/golang-lru"
+	lru "github.com/hashicorp/golang-lru/v2"
 	"github.com/symfony-cli/cert"
 )
 
 type certStore struct {
 	proxyCfg *Config
 	ca       *cert.CA
 	lock     sync.Mutex
-	cache    *lru.ARCCache
+	cache    *lru.ARCCache[string, tls.Certificate]
 }
 
 // newCertStore creates a store to keep SSL certificates in memory
 func (p *Proxy) newCertStore(ca *cert.CA) *certStore {
-	cache, _ := lru.NewARC(1024)
+	cache, _ := lru.NewARC[string, tls.Certificate](1024)
```

Comment:
Go generics!
I have never seen an actual use of them since they were introduced 😅